src/components/main/TrackListItem.tsx

```diff
@@ -0,0 +1,109 @@
+import styled from 'styled-components';
+import { ReactComponent as TrackDownSvg } from '../../assets/icons/mp3-download-icon.svg';
+import { ReactComponent as ScoreDownSvg } from '../../assets/icons/score-download-icon.svg';
+
+export type Track = {
+  track: {
+    id: number;
+    title: string;
+    vocal?: string;
+    songWriter?: string;
+    originalSong?: string;
+    arrangers?: string[];
+    soundTrackLink: string;
+    scoreLink?: string;
+  };
+};
+
+export const TrackListItem = ({ track }: Track) => {
```

Comment:
타입 Track 안에 track 필드가 있으면 말이 안되지
차라리 아래가 말이 될듯

```suggestion
type Track = {
  id: number;
  title: string;
  vocal?: string;
  songWriter?: string;
  originalSong?: string;
  arrangers?: string[];
  soundTrackLink: string;
  scoreLink?: string;
};

export type TrackListItemProps = {
  track: Track;
};

export const TrackListItem = ({ track }: TrackListItemProps) => {

Reply:
이거 props가 너무 많기두 하고... 애초에 데이터 객체로 받아오면 한 번에 객체를 넘겨서 받으려구 했던건데 호호... 내가 봐도 너무 끼워맞추는 식의 코드라구 할 수 있죠 음하하..